inst/schema/spec.md

```diff
@@ -3,6 +3,10 @@
 This API is built on top of [`pkgapi`](https://reside-ic.github.io/pkgapi) (itself influenced by [`hintr`](https://github.com/mrc-ide/hintr) and [montagu api](https://github.com/vimc/montagu-api/blob/master/spec/spec.md)).
 
 
+## GET /
+
+Index of available endpoints, for compatibility only and is not updated.
```

Comment:
@richfitz this is based on note in `api.R` what is this endpoint for? Should it be updated?